pages/common/units.md

```diff
@@ -0,0 +1,15 @@
+# units
+
+> Provide the conversion between two units of measure.
+
```

Comment:
Add this line in a sub-heading-  `Typing 'search {{text}}' in the prompt  will display a list of all of the units containing {{text}}.`

Reply:
Looks like that is the behavior of GNU units but not the units that is installed by default on, e.g. OS X. How do you want to handle the difference?

Reply:
Ah, in that case, you need to create separate copies of the page inside linux, windows and osx folders.